Squirrely-Ducks/bangazon-sprint-3
# User search

## DESCRIPTION
What did you do? 
When a user searches for a product in our navigation if the product they are looking for is available, then it will display more information regarding the search
Why did you do it?
This is a requirement of the project 
## STEPS TO TEST
1. npm start
2. login, and go to search bar and enter in a product example: Frozen Bacon
3. Test functionality
	* Search
	* Make sure product is displaying info
	* 

## REFERENCE
Does it reference a ticket?
